spec/index.html

```diff
@@ -1033,6 +1033,25 @@ <h4>Patterns of RDF entailment (Informative)</h4>
       <p><code>ex:a ex:p _:x . <br/>
        _:x rdf:type xsd:integer . </code></p>
 
+      <p>The last semantic condition above also justifies the following entailment pattern:</p>
+
+      <table>
+        <tbody>
+          <tr>
+            <th > </th>
+            <th ><strong>any S</strong></th>
+            <th ><strong>then S RDF entails, recognizing D</strong></th>
+          </tr>
+          <tr >
```

Comment:
Just dropping some unnecessary space chars
```suggestion
            <th> </th>
            <th><strong>any S</strong></th>
            <th><strong>then S RDF entails, recognizing D</strong></th>
          </tr>
          <tr>
```